Client/multiplayer_sa/CMultiplayerSA.cpp

```diff
@@ -578,6 +578,76 @@ CMultiplayerSA::CMultiplayerSA()
     m_dwLastStaticAnimID = eAnimID::ANIM_ID_WALK;
 }
 
+namespace GrainEffect
```

Comment:
I'd suggest also moving this into a separate file. CMultiplayerSA.cpp is long enough as-is